utils/tests_fetcher.py

```diff
@@ -1193,9 +1193,9 @@ def create_test_list_from_filter(full_test_list, out_path):
         default=None,
     )
     parser.add_argument(
-        "--commit_message",
```

Comment:
this is never used.

(The actual used one is `commit_message = repo.head.commit.message`)